<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>unnamed - Text_edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ing.qrc">
    <normaloff>:/Icons/icons8-text_edit-48.png</normaloff>:/Icons/icons8-text_edit-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="centr">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Create">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Создать&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string comment="Создать"/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-create new-26.png</normaloff>:/Icons/icons8-create new-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Open">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Открыть&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-open-26.png</normaloff>:/Icons/icons8-open-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Save">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сохранить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-save-26.png</normaloff>:/Icons/icons8-save-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_copy">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Копировать&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-copy-64.png</normaloff>:/Icons/icons8-copy-64.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cut">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вырезать&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-cut-32.png</normaloff>:/Icons/icons8-cut-32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_insert">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вставить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8 insert-32.png</normaloff>:/Icons/icons8 insert-32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Undo">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Отменить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-Undo-26.png</normaloff>:/Icons/icons8-Undo-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Do">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Повторить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-do-26.png</normaloff>:/Icons/icons8-do-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_font">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Изменить шрифт&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-F-26.png</normaloff>:/Icons/icons8-F-26.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Close">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Закрыть&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-close-32.png</normaloff>:/Icons/icons8-close-32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Exit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выход&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="ing.qrc">
            <normaloff>:/Icons/icons8-Exit-32.png</normaloff>:/Icons/icons8-Exit-32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-create new-26.png</normaloff>:/Icons/icons8-create new-26.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-open-26.png</normaloff>:/Icons/icons8-open-26.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-save-26.png</normaloff>:/Icons/icons8-save-26.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_5">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-save_as-24.png</normaloff>:/Icons/icons8-save_as-24.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-close-32.png</normaloff>:/Icons/icons8-close-32.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-Exit-32.png</normaloff>:/Icons/icons8-Exit-32.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-copy-64.png</normaloff>:/Icons/icons8-copy-64.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-cut-32.png</normaloff>:/Icons/icons8-cut-32.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8 insert-32.png</normaloff>:/Icons/icons8 insert-32.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-Undo-26.png</normaloff>:/Icons/icons8-Undo-26.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-do-26.png</normaloff>:/Icons/icons8-do-26.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-F-26.png</normaloff>:/Icons/icons8-F-26.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать шрифт</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/icons8-inf-26.png</normaloff>:/Icons/icons8-inf-26.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="icon">
    <iconset resource="ing.qrc">
     <normaloff>:/Icons/Qt.png</normaloff>:/Icons/Qt.png</iconset>
   </property>
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ing.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Create</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_3</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Open</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Save</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_6</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Close</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_7</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Exit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_8</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_copy</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_9</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Cut</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_10</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_insert</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_11</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Undo</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_12</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_Do</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_13</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_font</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
